tests/Unit/CallPowerTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use Chompy\Services\Rogue;
+
+class CallPowerTest extends TestCase
+{
+    /**
+     * Test that a post with a completed status successfully sends to Rogue.
+     *
+     * @return void
+     */
+    public function testCompletedCallStatus()
+    {
+        // Create a post with completed as the status.
+        $response = $this->withChompyApiKey()->json('POST', 'api/v1/callpower/call', [
+            'mobile' => '1234567891',
+            'callpower_campaign_id' => '1',
+            'status' => 'completed',
+            'call_timestamp' => '2017-11-09 06:34:01.185035',
+            'call_duration' => 50,
+            'campaign_target_name' => 'Mickey Mouse',
+            'campaign_target_title' => 'Representative',
+            'campaign_target_district' => 'FL-7',
+            'callpower_campaign_name' => 'Test',
+            'number_dialed_into' => '+12028519273',
+        ]);
+
+        $response->assertStatus(200);
+    }
+
+    /**
+     * Test that a post with a busy status successfully sends to Rogue.
+     *
+     * @return void
+     */
+    public function testBusyCallStatus()
+    {
+        // Create a post with completed as the status.
+        $response = $this->withChompyApiKey()->json('POST', 'api/v1/callpower/call', [
+            'mobile' => '1234567891',
+            'callpower_campaign_id' => '1',
+            'status' => 'busy',
+            'call_timestamp' => '2017-11-09 06:34:01.185035',
+            'call_duration' => 50,
+            'campaign_target_name' => 'Mickey Mouse',
+            'campaign_target_title' => 'Representative',
+            'campaign_target_district' => 'FL-7',
+            'callpower_campaign_name' => 'Test',
+            'number_dialed_into' => '+12028519273',
+        ]);
+
+        $response->assertStatus(200);
+    }
```

Comment:
@DFurnes still trying to wrap my head around what we are testing here! Is this on the right path?